packages/snage/src/provider/git-date.test.ts

```diff
@@ -0,0 +1,65 @@
+import {providerFactory} from './git-date';
+import * as E from 'fp-ts/lib/Either';
+import * as TE from 'fp-ts/lib/TaskEither';
+import semver from 'semver/preload';
+import {ValueProvider} from './provider';
+import path from 'path';
+import {assertRight} from '../fp/fp';
+import {LocalDate} from '@js-joda/core';
+
+describe('git-version', () => {
+    describe('providerFactory', () => {
+        it('fails when version-regex is missing', () => {
+            const provider: E.Either<string, ValueProvider> = providerFactory({
+                name: 'date',
+                type: 'date',
+                provided: {
+                    by: 'git-version',
+                },
+            });
+            expect(provider).toMatchObject(E.left(`Required argument 'version-tag' is missing.`));
+        });
+    });
+    describe('run without version', () => {
+        it('returns undefined', async () => {
+            const provider: E.Either<string, ValueProvider> = providerFactory({
+                name: 'date',
+                type: 'date',
+                provided: {
+                    by: 'git-version',
```

Comment:
Shouldn't this be git-date? It seems like a lot git-version is used in this file (:.

Reply:
Good catch. :)